### PR TITLE
Tutorial enhancement to keep it clean on docs.tvm.ai

### PR DESCRIPTION
**Pull request by srkreddy1238**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

**Comment by srkreddy1238** [comment]
@masahi @siju-samuel @PariksheetPinjari909 welcome to review pls.

File: tutorials/nnvm/from_tensorflow.py [hunk 1]
```diff
@@ -62,6 +82,13 @@
 ######################################################################
 # Decode image
 # ------------
+# .. note::
+#
+#   tensorflow frontend import doesn't support preprocessing ops like JpegDecode
+#   JpegDecode is a bypass (copy source) here.
```
Comment (masahi) [inline tutorials/nnvm/from_tensorflow.py]: is bypassed

File: tutorials/nnvm/from_tensorflow.py [hunk 2]
```diff
@@ -100,26 +142,28 @@ def transform_image(image):
...
     human_string = node_lookup.id_to_string(node_id)
     score = predictions[node_id]
     print('%s (score = %.5f)' % (human_string, score))
 
 ######################################################################
-# Run the same graph with tensorflow and dump output.
-# ---------------------------------------------------
+# Inference on tensorflow
+# -----------------------
+# Now lets run the same on tensorflow
```
Comment (masahi) [inline tutorials/nnvm/from_tensorflow.py]: run the corresponding model on tensorflow

File: tutorials/nnvm/from_tensorflow.py [hunk 3]
```diff
@@ -1,17 +1,20 @@
 """
 Compile Tensorflow Models
 =========================
-This article is an introductory tutorial to deploy tensorflow models with NNVM.
+This article is an introductory tutorial to deploy tensorflow models with TVM.
 
-For us to begin with, tensorflow module is required to be installed.
+For us to begin with, tensorflow python module is required to be installed.
 
 A quick solution is to install tensorlfow from
```
Comment (siju-samuel) [inline tutorials/nnvm/from_tensorflow.py]: tensorlfow > tensorflow

File: tutorials/nnvm/from_tensorflow.py [hunk 4]
```diff
@@ -47,8 +66,9 @@
...
 
 ######################################################################
-# Creates graph from saved graph_def.pb.
-# --------------------------------------
+# Import model
+# ------------
+# Creates tensorflow graph definition from protobuf file.
 
 with tf.gfile.FastGFile(os.path.join(
         "./", model_name), 'rb') as f:
```
Comment (siju-samuel) [inline tutorials/nnvm/from_tensorflow.py]: with tf.gfile.FastGFile(os.path.join("./", model_name), 'rb') as f:
> keep as single line

File: tutorials/nnvm/from_tensorflow.py [hunk 5]
```diff
@@ -62,6 +82,13 @@
...
 # ------------
+# .. note::
+#
+#   tensorflow frontend import doesn't support preprocessing ops like JpegDecode
+#   JpegDecode is a bypass (copy source) here.
+#   Hence we supply decoded frame to TVM instead.
+#
+
 from PIL import Image
 image = Image.open(img_name).resize((299, 299))
```
Comment (siju-samuel) [inline tutorials/nnvm/from_tensorflow.py]: `transform_image`
suggest to use `_transform_image` for private functions..

**Comment by srkreddy1238** [comment]
@masahi  & @siju-samuel thanks for the review. Handled the review comments now.

**Comment by tqchen** [comment]
Thanks @srkreddy1238 @masahi @PariksheetPinjari909 , this is now merged!

**Pull request merged**